dist/debian/build_deb.sh

```diff
@@ -90,11 +90,6 @@ PRODUCT=$(cat build/SCYLLA-PRODUCT-FILE)
 BUILDDIR=build/debian
 PACKAGE_NAME="$PRODUCT-machine-image"
 
-if echo $SCYLLA_VERSION | grep rc >/dev/null ; then
```

Comment:
shouldn't this be fixed at the source, i.e. in scylla repo ? or i'm missing something

Reply:
i.e. scylla itself saying the wrong version:
```
❯ /home/fruch/.ccm/scylla-repository/release/5.0.rc3/scylla/bin/scylla --version
5.0.rc3-0.20220406.f92622e0d
```

Reply:
The fix is only relevant from next time we will branch. following this issue (which was originally for docker images), we decided to align the pre-release so it will match both rpm and deb packages